src/test/java/org/folio/logintest/PasswordRepeatabilityValidationTest.java

```diff
@@ -117,9 +114,13 @@ public static void teardown(TestContext context) {
 
   @Test
   public void testNewPassword() {
+    Password passwordEntity = new Password()
```

Comment:
Please create separate method for _passwordEntity_

Reply:
done